web/styles/dark_theme.css

```diff
@@ -89,7 +89,9 @@
         box-shadow: inset 0 1px 0 hsl(0deg 0% 0% / 20%);
     }
 
-    .enter_sends_choices {
+    .enter_sends_choices,
+    #message-formatting,
+    #keyboard-shortcuts {
```

Comment:
If you're up for it, a better option here would be to move the light and dark theme colors to CSS variables, and just remove all of this extra complexity from `dark_theme.css`. Otherwise, it's just a cat-and-mouse game of forever fiddling with selectors in this file, as this PR does...

Reply:
Sure. I will do that.

Reply:
Overall this looks to be on the right track. I'm not certain about all the many colors introduced in the `app_variables.css` file in a single commit, but ideally those would be introduced in smaller, easier-to-review commits with their actual use in the CSS. That will make this easier to review, and likely easier to catch subtle regressions that can pop up in this kind of work.